API_TRANSITION_GUIDE.md

```diff
@@ -0,0 +1,62 @@
+## Deprecated API Transition Guide
+
+When API's are deprecated, the following guide will show how to transition from removed API's to the new ones
+
+
+### Upgrading From 3.x to 4.0
+
+In the 3.2.0 release, a few API's were set to be deprecated in the 4.0 release.  With the release of 4.0.0,  those API's have been removed.
+
+#### Receiever
+
+The `Receiver` class has been removed.
+
+`Receiver.accept` should be transitioned to `HTTP.toEvent`
```

Comment:
Maybe an example would be useful here? I guess each takes the same parameters, but just so people can see it right away.

Reply:
added an example